pkgs/by-name/va/vapoursynth-wnnm/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  gcc12Stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  mkl,
+  vapoursynth,
+  zimg,
+}:
+
+gcc12Stdenv.mkDerivation (finalAttrs: {
+  pname = "vapoursynth-wnnm";
+  version = "unstable-2023-07-07";
+
+  srcs = [
+    (fetchFromGitHub {
+      owner = "WolframRhodium";
+      repo = "VapourSynth-WNNM";
+      rev = "a8977b4365841bb27c232383cd9a306f70ef9f99";
+      name = "${finalAttrs.pname}-source";
+      hash = "sha256-B4jvl+Lu724QofDbKQObdcpQdlb8KQ2szAp780J9SUY=";
+    })
+    (fetchFromGitHub {
+      owner = "vectorclass";
+      repo = "version2";
+      rev = "refs/tags/v2.02.01";
+      name = "vectorclass";
+      hash = "sha256-45qt0vGz6ibEmcoPZDOeroSivoVnFkvMEihjXJXa8lU=";
+    })
```

Comment:
You might consider packaging this separately, especially if used by other packages.

Reply:
Currently the only packages that use this repo are some of the vapoursynth plugins I packaged. Not sure how useful that would be for other people. If I encounter more packages using `vectorclass` I will package it separately.